packages/components/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
```

Comment:
These are already present (commented out) down in the "Modules" section if we were continuing to roll with the structured form of key/values.